### ISSUE TITLE
Review GroupRequiredMixin

### ISSUE THREAD

**Issue by cdobbyn** [BCGamer/website#166]
This might still apply after Mezzanine has been replaced.

Example: salesbro.views.VendorLogon()

If it fails validation the error doesn't match the problem encountered. Not a major issue for now. The check for logon / user group still occurs.

**Comment by cdobbyn** [comment]
Moving to on hold, to check after #285 is complete.

**Comment by cdobbyn** [comment]
Wrong issue closed.

**Comment by cdobbyn** [comment]
This isn't a problem outside of Mezzanine. Closing.
